static/app/components/replays/replayController.tsx

```diff
@@ -54,10 +54,10 @@ function ReplayPlayPauseBar({isCompact}: {isCompact: boolean}) {
       {!isCompact && (
```

Comment:
That's kinda one of the reasons I was asking about the updated design it doesn't really seem like it would need this since it is already pretty compact

Reply:
Yeah my thinking as well! Maybe we can tidy this up when you've finished the settings composite menu PR?

Reply:
I think I kinda raised this with Ryan already and it sounded like maybe there was still a use case so maybe I didn't do it for nothing but we'll see :)